src/main/java/org/spongepowered/api/data/manipulator/generator/testing/DummyCustomDataData.java

```diff
@@ -0,0 +1,45 @@
+package org.spongepowered.api.data.manipulator.generator.testing;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.data.DataQuery;
+import org.spongepowered.api.data.key.Key;
+import org.spongepowered.api.data.key.KeyFactory;
+import org.spongepowered.api.data.manipulator.DataManipulatorBuilder;
+import org.spongepowered.api.data.manipulator.generator.DataRegistration;
+import org.spongepowered.api.data.meta.ItemEnchantment;
+import org.spongepowered.api.data.value.mutable.ListValue;
+import org.spongepowered.api.data.value.mutable.MutableBoundedValue;
+import org.spongepowered.api.entity.living.Agent;
+import org.spongepowered.api.util.CommonTokens;
+
+public class DummyCustomDataData {
```

Comment:
Clearly you missed the point of "WIP" and the last time that I've updated this PR... There's other PR's/branches that have been made since towards this overall PR.